### PR TITLE
[bitnami/kibana] set base host to 0.0.0.0

### LINKED ISSUE #17375: [FATAL][root] Error: listen EAFNOSUPPORT: address family not supported :::5601
### Name and Version

https://github.com/bitnami/charts/tree/main/bitnami/kibana/#installing-the-chart

### What architecture are you using?

None

### What steps will reproduce the bug?

Run

### Are you using any custom parameters or values?

helm install kibana oci://registry-1.docker.io/bitnamicharts/kibana \
  --set service.type=NodePort \
  --set global.storageClass=nfs-client \
  --set persistence.storageClass=nfs-client \
  --set service.nodePorts.http=30030 \
  --set elasticsearch.hosts[0]=elasticsearch-master \
  --set elasticsearch.port=9200

### What is the expected behavior?

_No response_

### What do you see instead?

[FATAL][root] Error: listen EAFNOSUPPORT: address family not supported :::5601

### Additional information

Hello
This error occurs after installing in Logpod

### PR DESCRIPTION
**Pull request by mike12806**
### Description of the change

This change defaults the host value to 0.0.0.0 instead of `::` in the configmap because unless folks have ipv6 enabled, :: causes Kibana to fail on startup. 

### Benefits

The majority of the world is using IPV4 and this is a more sane default. If needed, folks can update for ipv6 compatability manually. 

### Possible drawbacks

If you're using ipv6 without any ipv4 fallback, this change may be breaking (unlikely scenario) 

### Applicable issues

- fixes #17375

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute to this project. Please take a look at my comments

File: bitnami/kibana/templates/configmap.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ metadata:
 data:
   kibana.yml: |
     pid.file: /opt/bitnami/kibana/tmp/kibana.pid
-    server.host: "::"
+    server.host: "0.0.0.0"
```
[the remark this answers is not in the document]
Comment (mike12806) [reply bitnami/kibana/templates/configmap.yaml:19]: Yeah I could do that - so you're thinking have it be a parameter under configuration.server, maybe with a default value of "0.0.0.0"? I like that

File: bitnami/kibana/templates/configmap.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ metadata:
 data:
   kibana.yml: |
     pid.file: /opt/bitnami/kibana/tmp/kibana.pid
-    server.host: "::"
+    server.host: {{ .Values.configuration.server.host }}
```
Comment (jotamartos) [inline bitnami/kibana/templates/configmap.yaml:19]:
```suggestion
    server.host: {{ .Values.configuration.server.host | quote }}
```

**Review by jotamartos — CHANGES_REQUESTED** [review]
Please take a look at the minor suggestion I added and fix the DCO issue

**Comment by mike12806** [comment]
@jotamartos I'll bring these changes into a new PR / branch in a few days as my re-base here got a bit messy trying to fix the DCO issue.

**Comment by mike12806** [comment]
Closing in favor of https://github.com/bitnami/charts/pull/30131

**Pull request closed without merging**